pages/docs/tutorials/native/working-with-klib.md

```diff
@@ -0,0 +1,140 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Working with Kotlin/Native Libraries"
+description: "A look at how to work with Kotlin/Native libraries"
+authors: Hadi Hariri 
+date: 2018-01-22
+showAuthorInfo: false
+---
+
+
+In this tutorial we'll see
+
+* [What is a Kotlin/Native Libary](#what-is-kotlin/native-library)
+* [How to create a Kotlin/Native Library](#creating-a-kotlin/native-libraries)
+* [How to consume Kotlin/Native libraries](#consuming-a-kotlin/native-library)
+* [How to use and create library repositories](#using-and-creating-library-repositories)
+
+
+## What is Kotlin/Native Library
+
+The concept of a library, a collection of resources to be used by applications, is available in Kotlin/Native and it is represented
```

Comment:
"Resources" is usually used to mean "not code". A library contains code. Better to change this to "functions" or "APIs" or something like that.

Reply:
Well technically resources can mean code also, and the reason I said resources is because the library can contain images and other things. But I'll rephrase it somewhat.